css/style.css

```diff
@@ -0,0 +1,170 @@
+
+.backg-1{
+    background-image: url("bg1.png");
+    background-size:contain;
+    background-repeat: no-repeat;
+    margin-left: 0;
+}
+.btn{
+    font-size: 17px;
+    width: 180px;
+    margin: 10px;
+    padding: 10px;
+    line-height: 2;
+    border-radius: 55px;
+    
+}
+.linija1 {
+    border: none;
+    background-color: rgb(0, 0, 0);
+    width: 50px;
+    transform: scaleX(0.5); 
+    transform: scaleY(3); 
+}
+#edwardbcg{
+        background: linear-gradient(208.18deg, #67C3F3 9.05%, #5A98F2 76.74%);
+        border-radius: 24px;
+        height: 425px;
+        width: 1120px;     
+}
+.linija2{
+    border: none;
+    background-color: white;
+    width: 60px;
+    margin: auto;
+    transform: scaleY(3); 
+}
+
+.fotter{
+    background: linear-gradient(183.41deg, #67C3F3 -8.57%, #5A98F2 82.96%);
+}
+
+.logo{
+    height: 41px;
+    width: 41px;
+}
+
+.backg-2{
+    background-image: url("element.png");
+    background-repeat: no-repeat;
+    background-position: right;
+}
+
+.bord2{
+    height: 512px;
+    width: 350px;
+    border:0  solid rgb(82, 82, 82);
+}
+
+.shadow1{
+    box-shadow: 20px 10px 25px rgba(0,0, 0, 0.3);
+}
+
+ul{
+    list-style: none;
+}
+
+.dots-top{
+    position: absolute;
+    margin-top: 30px;
+    margin-left: -100px;
+    opacity: 30%;
+}
+
+.dots-our{
+    opacity: 30%;
+    position: absolute;
+    margin-left: 1390px;
+    margin-top: 500px;
+    margin-right: 0;
+    
+}
+.dots-fot1{
+    opacity: 30%;
+    position: absolute;
+    margin-left: 1550px;
+    margin-top: -80px;
+    z-index: -1;
+}
+.dots-fot{
+    height: 140px;
+    position: absolute;
+    margin-top: 300px;
+    margin-left: 20px;
+    opacity: 30%;
+}
+
+.dots-art{
+    height: 140px;
+    position: absolute;
+    margin-top: 150px;
+    margin-left: 250px;
+    opacity: 30%;
+}
+.dots-customers{
+    height: 140px;
+    position: absolute;
+    margin-top: 300px;
+    margin-left: 330px;
+    opacity: 30%;
+}
+.dots-customers1{
+    height: 120px;
+    position: absolute;
+    margin-top: 60px;
+    margin-left: 1390px;
+    z-index: 1;
+    opacity: 30%;
+    
+}
+
+.carousel-indicators [data-bs-target] {
+    border-radius: 50%;
+    width: 15px;
+    height: 15px;
+    background-color: #5A98F2;
+}
+.carousel-indicators{
+    top: 430px;
+
+}
+.carousel-control-next-icon {
+    background-image:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%2367C3F3'%3e%3cpath d='M4.646 1.646a.5.5 0 0 1 .708 0l6 6a.5.5 0 0 1 0 .708l-6 6a.5.5 0 0 1-.708-.708L10.293 8 4.646 2.354a.5.5 0 0 1 0-.708z'/%3e%3c/svg%3e")
+}
+.carousel-control-prev-icon {
+    background-image:url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%2367C3F3'%3e%3cpath d='M11.354 1.646a.5.5 0 0 1 0 .708L5.707 8l5.647 5.646a.5.5 0 0 1-.708.708l-6-6a.5.5 0 0 1 0-.708l6-6a.5.5 0 0 1 .708 0z'/%3e%3c/svg%3e")
+    
+}
+.carousel-control-prev,
+.carousel-control-next
+ {
+    top: 450px;

```

Comment:
![image](https://github.com/JurasLeo/Zadatak1/assets/45786578/c7b7d8cb-a3ab-4d67-bedb-27e786152f03)
maybe nešto ovako ili posebno namjestiti prev i next opciju za left/right, ali generalno je dobro za prvi put :)